src/system/SystemPacketBuffer.h

```diff
@@ -649,7 +649,7 @@ inline PacketBufferHandle PacketBuffer::Next()
 inline PacketBufferHandle PacketBuffer::Last()
 {
     PacketBuffer * p = this;
-    while (p->Next_ForNow() != nullptr)
+    while (p->next != nullptr)
```

Comment:
Should stay `Next_ForNow()`.

Reply:
Done. (For my next cleanup PR, I'm currently intending a private `ChainedBuffer()` following the naming of the existing `HasChainedBuffer()` that replaced all the `next == nullptr` last time.)